net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Client.xml

```diff
@@ -14,7 +14,7 @@
                     <Required>Y</Required>
                 </name>
                 <pubkey type="TextField">
-                    <Required>N</Required>
+                    <Required>Y</Required>
                 </pubkey>
                 <psk type="TextField">
```

Comment:
Same here, `TextField` should be changed for `Base64Field` and mask should then be either updated or removed depending on what's better

Reply:
I thought you would provide an updated URL?